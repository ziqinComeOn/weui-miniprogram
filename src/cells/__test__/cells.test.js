import path from 'path'
import simulate from 'miniprogram-simulate'

describe('cells & cell', () => {
    let id

    beforeAll(() => {
        id = simulate.load(path.resolve(__dirname, './index'), { less: true })
    })

    test('basic cells & cell', async () => {
        const container = simulate.render(id)
        container.attach(document.createElement('parent-wrapper'))
        expect(container.toJSON()).toMatchSnapshot()
    })
})
