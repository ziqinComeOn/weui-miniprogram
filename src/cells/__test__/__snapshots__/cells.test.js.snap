// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`cells & cell basic cells & cell 1`] = `
<main>
  <mp-cells
    class="cells"
  >
    <wx-view
      class=" weui-cells__group  "
    >
      <wx-view
        class="weui-cells__title"
      >
        带图标、说明的列表项
      </wx-view>
      <wx-view
        class="weui-cells weui-cells_after-title "
      >
        <mp-cell
          class="cell1"
        >
          <wx-view
            class="weui-cell    "
            hoverClass=""
            bind:tap="navigateTo"
          >
            <wx-view
              class="weui-cell__hd "
            >
              <wx-image
                class="weui-cell__icon"
                mode="aspectFit"
                src="success"
              />
            </wx-view>
            <wx-view
              class="weui-cell__bd "
            >
              标题文字
            </wx-view>
            <wx-view
              class="weui-cell__ft "
            >
              说明文字
            </wx-view>
          </wx-view>
        </mp-cell>
        <mp-cell
          class="cell2"
        >
          <wx-view
            class="weui-cell    weui-cell_wxss"
            hoverClass=""
            bind:tap="navigateTo"
          >
            <wx-view
              class="weui-cell__hd "
            />
            <wx-view
              class="weui-cell__bd "
            >
              标题文字
            </wx-view>
            <wx-view
              class="weui-cell__ft "
            >
              说明文字
            </wx-view>
          </wx-view>
        </mp-cell>
      </wx-view>
      <wx-view
        class="weui-cells__tips"
      >
        底部说明文字
      </wx-view>
    </wx-view>
  </mp-cells>
</main>
`;
